tests/test_transforms.py

```diff
@@ -8,71 +8,127 @@
         {"param_array_3": jnp.asarray(np.linspace(-10, 10, 4))},
     ]
 
-    tf = ParamTransform(lowers, uppers)
-
-    assert all(tf.forward(params)[0]["param_array_1"] > lowers["param_array_1"])
-    assert all(tf.forward(params)[0]["param_array_1"] < uppers["param_array_1"])
-    assert any(
-        tf.forward(params)[1]["param_array_2"] < lowers["param_array_1"]
-    )  # lower not constrained
-    assert all(tf.forward(params)[1]["param_array_2"] < uppers["param_array_2"])
-    assert all(tf.forward(params)[2]["param_array_3"] > lowers["param_array_3"])
+    tf = ParamTransform(tf_dict)
+    forward = tf.forward(params)
+
+    assert all(
+        forward[0]["param_array_1"] > lowers["param_array_1"]
+    ), "SigmoidTransform failed to match lower bound."
+    assert all(
+        forward[0]["param_array_1"] < uppers["param_array_1"]
+    ), "SigmoidTransform failed to match upper bound."
+    assert all(
+        forward[1]["param_array_2"] < uppers["param_array_2"]
+    ), "SoftplusTransform failed to match upper bound."
+    assert all(
+        forward[2]["param_array_3"] > lowers["param_array_3"]
+    ), "NegSoftplusTransform failed to match lower bound."
     assert any(
-        tf.forward(params)[2]["param_array_3"] > uppers["param_array_1"]
+        forward[2]["param_array_3"] > uppers["param_array_1"]
     )  # upper not constrained
 
 
-def test_jit():
+@pytest.mark.parametrize(
+    "transform",
+    [
+        jt.SigmoidTransform(-2, 2),
+        jt.SoftplusTransform(2),
+        jt.NegSoftplusTransform(2),
+        jt.AffineTransform(1.0, 1.0),
+        jt.CustomTransform(lambda x: x, lambda x: x),
+        jt.ChainTransform([jt.SigmoidTransform(-2, 2), jt.SoftplusTransform(2)]),
+    ],
+)
+def test_jit(transform):
     # test jit-compilation:
-    lowers = {"param_array_1": 2}
-    uppers = {"param_array_1": -2}
+    tf_dict = {"param_array_1": transform}
 
     params = [{"param_array_1": jnp.asarray(np.linspace(-1, 1, 4))}]
 
-    tf = ParamTransform(lowers, uppers)
+    tf = ParamTransform(tf_dict)
 
     @jit
     def test_jit(params):
         return tf.inverse(params)
 
     _ = test_jit(params)
+
+
+@pytest.mark.parametrize(
+    "transform",
+    [
+        jt.SigmoidTransform(-2, 2),
+        jt.SoftplusTransform(2),
+        jt.NegSoftplusTransform(2),
+        jt.AffineTransform(1.0, 1.0),
+        jt.CustomTransform(lambda x: x, lambda x: x),
+        jt.ChainTransform([jt.SigmoidTransform(-2, 2), jt.SoftplusTransform(2)]),
+    ],
+)
+def test_correct(transform):
+    # test jit-compilation:
+    tf_dict = {"param_array_1": transform}
+
+    params = [{"param_array_1": jnp.asarray(np.linspace(-1, 1, 4))}]
+
+    tf = ParamTransform(tf_dict)
+
+    forward = tf.forward(params)
+    inverse = tf.inverse(forward)
+
+    assert np.allclose(
+        inverse[0]["param_array_1"], params[0]["param_array_1"]
+    ), f"{transform} forward, inverse failed."
```

Comment:
Awesome! Can we have some more elaborate test that checks whether it plays well with `.get_parameters()`? 

E.g.
```python
comp = jx.Compartment()
branch = jx.Branch(comp, nseg=2)
cell = jx.Cell(branch, parents=[-1, 0, 0])

cell.branch("all").make_trainable("radius")
cell.branch(2).make_trainable("radius")
cell.branch(1).make_trainable("length")
cell.branch(0).make_trainable("radius")
cell.branch("all").comp("all").make_trainable("axial_resistivity")
cell.make_trainable("capacitance")

transform = ...

params = cell.get_parameters()
transformed_params = transform.forward(params)
recovered_params = transform.inverse(transformed_params)
# ...test that they are still the same, or that transformed_params has no NaN.
```

Reply:
Added it.

I do scale everything by 1/1000 (as axial resistivity is 5000 for which most transforms are not numerically invertible)